kubernetes/deployment.yml

```diff
@@ -23,7 +23,13 @@ spec:
             path: config.yml
```

Comment:
You mean that the configuration should be passed via the command line in the `deployment.yml` directly?